TinyOAuth1/AccessTokenInfo.cs

```diff
@@ -1,8 +1,11 @@
+using System.Collections.Generic;
+
 namespace TinyOAuth1
 {
 	public class AccessTokenInfo
 	{
 		public string AccessToken { get; set; }
 		public string AccessTokenSecret { get; set; }
+	    public IDictionary<string, string> AdditionalParams { get; set; }
```

Comment:
Formatting.

Reply:
Yeah, I don't see it in my editor. Which is odd, because it isn't a tabs/spaces thing I don't think. Will fix.

Reply:
Ah. It is. You use tabs.